common/ethnicities/wc_ethnicities_elven.txt

```diff
@@ -370,7 +370,7 @@ high_elven = {
 	
 	#Eyes
 	special_eyes = {
-        1 = { name = magic_eyes      range = { 1 1 } }
+        1 = { name = magic_eyes      range = { 0.5 0.5 } }
```

Comment:
The glow of the DNA elves Kael'thas, Anasterian has not changed. Is it inteted?

Reply:
Wasn't intended. Fixed in 617d9e53e1df8cb0acac34d0f0790dbcf08923f1.